collections/_infographics/policies/fit-for-55.md

```diff
@@ -1,13 +1,134 @@
+
+#### Transportation
+
+<!--* [Stricter emission regulations for cars and vans](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0556)-->
+* [New infrastructure for alternative fuels](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0559)
+* [Initiative for more sustainable aviation fuels ReFuelEU](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0561)
+* [FuelEU Cleaner Marine Fuels Initiative](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0562)
+
+#### Landscape and ecosystems
+
+* [EU Forest Strategy](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021DC0572)
+* [Revision of the Land Use and Forestry Regulation](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0554)
+
+### Support measures
+
+* Establishment of the [Social Climate Fund](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0568)
+* Strengthening [Modernization](https://ec.europa.eu/clima/eu-action/funding-climate-action/modernisation-fund_en) and [Innovation Fund](https://ec.europa.eu/clima/eu-action/funding-climate-action/innovation-fund_en)
+<!--
+For more information on the proposed measures, go to our infographic [Fit for 55: an overview of the proposed measures](/infografiky/fit-for-55-opatreni).
+-->
+
+## Resources and notes
+
+### Who to contact next
+
+Here are some of the experts who deal with the 'Fit for 55' package:
+
+[Kateřina Davidová](https://www.cde-org.cz/cs/kontakt), Centre for Transport and Energy  
+[Štěpán Vizi](https://www.cde-org.cz/cs/kontakt), Centre for Transport and Energy  
+[Michal Hrubý](https://www.europeum.org/user/default/132), EUROPEUM Institute for European Policy  
+[Tomáš Trubačík](https://sanceprobudovy.cz/o-nas/), Opportunities for buildings  
+[Tomáš Jungwirth](https://www.amo.cz/autor/tomas-jungwirth/), Association for International Affairs
+
+### Resources
+
+[^fit-for-55]: ['Fit for 55': delivering the EU's 2030 Climate Target on the way to climate neutrality](https://eur-lex.europa.eu/legal-content/en/TXT/?uri=CELEX%3A52021DC0550)
+
+[^presidency-eu]: [Czech Presidency of the Council of the European Union](https://czech-presidency.consilium.europa.eu/en/)
+
+[^fit-for-55-q-a]: Carbon Brief: [Q&A: How ‘Fit for 55’ reforms will help EU meet its climate goals](https://www.carbonbrief.org/qa-how-fit-for-55-reforms-will-help-eu-meet-its-climate-goals)
+
+[^emission-co2]: Our World in Data: [CO<sub>2</sub> emissions by region](https://ourworldindata.org/co2-emissions#co2-emissions-by-region)
+
+[^eurobarometer]: [Eurobarometer 2021 ](https://climate.ec.europa.eu/system/files/2021-06/cz_climate_2021_en.pdf). The percentage of those who perceive climate change as a serious problem includes respondents who perceive it as a *very* or *somewhat* serious problem. Percentage agreeing with the statement includes respondents who *strongly* or *likely* agree with the statement.
```

Comment:
Add "The" in front of the second percentage. "The percentage agreeing with"...

Reply:
Thanks. Updated the article here.